sdk/src/main/java/ly/count/android/sdk/ConnectionQueue.java

```diff
@@ -388,13 +388,6 @@ public void sendUserData(String userdata) {
             return;
         }
 
-        if (userdata.equals("")) {
-            L.d("[Connection Queue] No user data to send, skipping");
-            return;
-        }
-
-        moduleRequestQueue.sendEventsIfNeeded(true); // flush events before sending user details //todo this should be moved to the user profile modile after removing the static user profile implementation
```

Comment:
this is also not needed to be called here because in other places it is called, and it is not its place to be called